### PR TITLE
issue #450, edits to save work form

### PR DESCRIPTION
**Pull request by surreal8**
Hi Adam, this is almost complete, could you add the visibility options for public and authenticated in the helper files?

### CODE REVIEW

**Comment by awead** [comment]
@surreal8 did you test the form submission with these changes?

**Comment by surreal8** [comment]
@awead yes

**Comment by awead** [comment]
@surreal8 you need to use the config option `Sufia.config.active_deposit_agreement_acceptance` instead. You won't need to change any of the javascript.

File: config/locales/sufia.en.yml [hunk 1]
```diff
@@ -166,3 +168,5 @@ en:
       department:
         text: "Department"
         label_html: "Department"
+      open:
+         label_html: Open Access
```
Comment (awead) [inline config/locales/sufia.en.yml:172]: something's wrong here. we shouldn't need to add these here, they're already a part of Sufia.

**Comment by awead** [comment]
@surreal8 see https://github.com/projecthydra/sufia/wiki/Customizing-Metadata#creating-a-default-deposit-agreement

File: config/locales/sufia.en.yml (hunk 1 above)
Comment (surreal8) [inline config/locales/sufia.en.yml:172]: Default label is "Public" I'm changing that here. Is this not the right place to override the label?

Comment (surreal8) [inline config/locales/sufia.en.yml:172]: actually these are all overrides of the default values

**Comment by surreal8** [comment]
thanks will take a look now at the agreement

**Comment by awead** [comment]
@surreal8 sorry, I was wrong about the JS. See https://github.com/projecthydra/sufia/issues/2385 but you shouldn't need to override the view html at least.

File: config/locales/sufia.en.yml (hunk 1 above)
Comment (awead) [inline config/locales/sufia.en.yml:172]: let me check into this. Open Access shows up other places, so I'm not sure what's going on.

**Comment by surreal8** [comment]
@awead ok, I've updated the pull request leaving the view html with the original code and editing the sufia.rb file

**Pull request merged**